tests/phpunit/payment-methods/class-wc-stripe-upe-payment-method-test.php

```
@@ -796,6 +796,72 @@ public function test_non_card_methods_are_not_available_when_optimized_checkout_
+	public function provide_non_deferred_intent_methods_available_in_oc_mode() {
+		return [
+			'BLIK with PLN currency' => [
+				'payment_method_class' => WC_Stripe_UPE_Payment_Method_BLIK::class,
+				'currency'             => WC_Stripe_Currency_Code::POLISH_ZLOTY,
+			],
+			'ACSS with CAD currency' => [
+				'payment_method_class' => WC_Stripe_UPE_Payment_Method_ACSS::class,
+				'currency'             => WC_Stripe_Currency_Code::CANADIAN_DOLLAR,
+			],
+		];
+	}
```

PHPMD (2.15.0)

[error] The method provide_non_deferred_intent_methods_available_in_oc_mode is not named in camelCase. (undefined)

(CamelCaseMethodName)